tests/standard_to_enhanced/planned_events_translator_test.py

```diff
@@ -0,0 +1,461 @@
+    assert test_feature == expected_feature
+
+
+def test_parse_work_zone_invalid_data():
+    test_var = 'a,b,c,d'
+    test_feature = planned_events_translator.parse_work_zone(test_var)
+    assert test_feature == None
+
+
+# --------------------------------------------------------------------------------Unit test for get_vehicle_impact function--------------------------------------------------------------------------------
+def test_get_vehicle_impact_some_lanes_closed():
+    lanes = [{
+        "order": 1,
+        "type": "shoulder",
+        "status": "open"
+    },
+        {
+        "order": 2,
+        "type": "general",
+        "status": "closed"
+    },
+        {
+        "order": 3,
+        "type": "general",
+        "status": "closed"
+    },
+        {
+        "order": 4,
+        "type": "shoulder",
+        "status": "open"
+    }]
+    test_vehicle_impact = planned_events_translator.get_vehicle_impact(
+        lanes)
+    expected_vehicle_impact = "some-lanes-closed"
+    assert test_vehicle_impact == expected_vehicle_impact
+
+
+def test_get_vehicle_impact_all_lanes_closed():
+    lanes = [{
+        "order": 1,
+        "type": "shoulder",
+        "status": "closed"
+    },
+        {
+        "order": 2,
+        "type": "general",
+        "status": "closed"
+    },
+        {
+        "order": 3,
+        "type": "general",
+        "status": "closed"
+    },
+        {
+        "order": 4,
+        "type": "shoulder",
+        "status": "closed"
+    }]
+    test_vehicle_impact = planned_events_translator.get_vehicle_impact(
+        lanes)
+    expected_vehicle_impact = "all-lanes-closed"
+    assert test_vehicle_impact == expected_vehicle_impact
+
+
+def test_get_vehicle_impact_all_lanes_open():
+    lanes = [{
+        "order": 1,
+        "type": "shoulder",
+        "status": "open"
+    },
+        {
+        "order": 2,
+        "type": "general",
+        "status": "open"
+    },
+        {
+        "order": 3,
+        "type": "general",
+        "status": "open"
+    },
+        {
+        "order": 4,
+        "type": "shoulder",
+        "status": "open"
+    }]
+    test_vehicle_impact = planned_events_translator.get_vehicle_impact(
+        lanes)
+    expected_vehicle_impact = "all-lanes-open"
+    assert test_vehicle_impact == expected_vehicle_impact
+
+
+# --------------------------------------------------------------------------------Unit test for wzdx_creator function--------------------------------------------------------------------------------
+@patch.dict(os.environ, {
+    'contact_name': 'Ashley Nylen',
+    'contact_email': 'ashley.nylen@state.co.us',
+    'issuing_organization': 'CDOT'
+})
+@patch('uuid.uuid4')
+@unittest.mock.patch('wzdx.standard_to_enhanced.navjoy_translator.datetime')
+@unittest.mock.patch('wzdx.tools.wzdx_translator.datetime')
+def test_wzdx_creator(mock_dt, mock_dt_3, mockuuid):
+    init_datetime_mocks([mock_dt, mock_dt_3])
+    uuid.uuid4 = Mock()
```

Comment:
If we're resetting the uuid.uuid4 to be a Mock here, do we need to patch it above?

Reply:
I can't really explain why, but this fails. If I run it with 
```
mockuuid.uuid4 = Mock()
mockuuid.uuid4.side_effect = 'we234de'
```
then it does not mock the call of uuid.uuidv4() in the called code. The only I could get it to work was with uuid.uuidv4

Reply:
Updated to use 
```
@patch.object(uuid, 'uuid4', side_effect=['w', '', '3'])
```
instead